impose/session.py

```diff
@@ -278,9 +279,11 @@ def find_file(name, search_paths, signature=None):
         else:
             raise ImposeDataFileNotFoundError(
                 name,
```

Comment:
This line should be removed, correct?

Reply:
I think so, yes. Just wasn't sure whether I should touch it or not. I will adjust the PR.

Reply:
> I think so, yes. Just wasn't sure whether I should touch it or not. I will adjust the PR.

Done.